docs/low-code-spark/gems/source-target/warehouse/cosmos.md

```diff
@@ -0,0 +1,134 @@
+| Schema Conversion Mode                            | `False`  | `Relaxed`     | The schema conversion behavior (`Relaxed`, `Strict`). When reading JSON documents, if a document contains an attribute that does not map to the schema type, the user can decide whether to use a null value (Relaxed) or an exception (Strict).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| Partitioning Strategy                             | `False`  | `Default`     | The partitioning strategy used (`Default`, `Custom`, `Restrictive` or `Aggressive`)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| Partitioning Targeted Count                       | `False`  | `None`        | The targeted Partition Count. This parameter is optional and ignored unless strategy is `Custom`. In this case the Spark Connector won't dynamically calculate the number of partitions but stick with this value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
integer value from ___ to ___.

Reply:
No more information is mentioned on the Connector Docs, so not sure if there IS in fact a range